bundles/core/src/main/java/com/adobe/cq/email/core/components/internal/models/EmailPageImpl.java

```diff
@@ -15,230 +15,7 @@
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 package com.adobe.cq.email.core.components.internal.models;
 
-
-import javax.annotation.PostConstruct;
-import javax.inject.Inject;
-
-import org.apache.sling.api.SlingHttpServletRequest;
-import org.apache.sling.api.resource.Resource;
-import org.apache.sling.models.annotations.Default;
-import org.apache.sling.models.annotations.Model;
-import org.apache.sling.models.annotations.Optional;
-import org.apache.sling.models.annotations.injectorspecific.Self;
-import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
-import org.apache.sling.models.factory.ModelFactory;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.adobe.cq.email.core.components.models.EmailPage;
-import com.adobe.cq.mcm.campaign.LinkingStatus;
-import com.adobe.cq.mcm.campaign.LinkingStatusService;
-import com.adobe.cq.mcm.campaign.NewsletterException;
-import com.adobe.cq.wcm.core.components.models.Page;
-import com.adobe.xfa.ut.StringUtils;
-import com.day.cq.i18n.I18n;
-import com.day.cq.mcm.campaign.ACConnectorException;
-import com.day.cq.mcm.campaign.CampaignConnector;
-import com.day.cq.mcm.campaign.CampaignException;
-import com.day.cq.mcm.campaign.NewsletterStatus;
-import com.day.cq.mcm.campaign.StatusService;
-import com.day.cq.wcm.api.WCMMode;
-import com.day.cq.wcm.webservicesupport.Configuration;
-
-/**
- * {@link EmailPage} implementation
- */
-@Model(
-    adaptables = SlingHttpServletRequest.class,
-    adapters = {EmailPage.class},
-    resourceType = EmailPageImpl.RESOURCE_TYPE
-)
-public class EmailPageImpl implements EmailPage {
-
+public class EmailPageImpl {
```

Comment:
This does not seem to be used anymore, could we remove it?

Reply:
The resourceType constant is used in multiple places. As it may be that we will have a EmailPage in the future, I thought keeping the class for now and use it to centrally declare the page's resourceType constant makes sense.